README.md

```diff
@@ -102,43 +126,17 @@ docker run -d --rm -e "MODE=OFFLINE" -e "NETWORK=TESTNET" -e "PORT=8081" -p 8081
-This instance type has 8 vCPU and 16 GB of RAM. If you use a computer with less than 16 GB of RAM,
-it is possible that `rosetta-ethereum` will exit with an OOM error.
 
-### Recommended OS Settings
-To increase the load `rosetta-ethereum` can handle, it is recommended to tune your OS
-settings to allow for more connections. On a linux-based OS, you can run the following
-commands ([source](http://www.tweaked.io/guide/kernel)):
-```text
-sysctl -w net.ipv4.tcp_tw_reuse=1
-sysctl -w net.core.rmem_max=16777216
-sysctl -w net.core.wmem_max=16777216
-sysctl -w net.ipv4.tcp_max_syn_backlog=10000
-sysctl -w net.core.somaxconn=10000
-sysctl -p (when done)
-```
-_We have not tested `rosetta-ethereum` with `net.ipv4.tcp_tw_recycle` and do not recommend
-enabling it._
-
-You should also modify your open file settings to `100000`. This can be done on a linux-based OS
-with the command: `ulimit -n 100000`.
 
 ## Testing with rosetta-cli
 To validate `rosetta-ethereum`, [install `rosetta-cli`](https://github.com/coinbase/rosetta-cli#install)
 and run one of the following commands:
-* `rosetta-cli check:data --configuration-file rosetta-cli-conf/testnet/config.json`
-* `rosetta-cli check:construction --configuration-file rosetta-cli-conf/testnet/config.json`
-* `rosetta-cli check:data --configuration-file rosetta-cli-conf/mainnet/config.json`
-
-## Future Work
-* Add ERC-20 Rosetta Module to enable reading ERC-20 token transfers and transaction construction
-* [Rosetta API `/mempool/*`](https://www.rosetta-api.org/docs/MempoolApi.html) implementation
-* Add more methods to the `/call` endpoint (currently only supports `eth_getBlockByNumber` and `eth_getTransactionReceipt`)
-* Add CI test using `rosetta-cli` to run on each PR (likely on a regtest network)
+* `rosetta-cli check:data --configuration-file rosetta-cli-conf/testnet/config.json` - This command validates that the Data API information in the `testnet` network is correct. It also ensures that the implementation does not miss any balance-changing operations.
+* `rosetta-cli check:construction --configuration-file rosetta-cli-conf/testnet/config.json` - This command validates the blockchain’s construction, signing, and broadcasting.
+* `rosetta-cli check:data --configuration-file rosetta-cli-conf/mainnet/config.json` - This command validates that the Data API information in the `mainnet` network is correct. It also ensures that the implementation does not miss any balance-changing operations.
 
-_Please reach out on our [community](https://community.rosetta-api.org) if you want to tackle anything on this list!_
+## Issues
+Interested in helping fix issues in this repository? You can find to-dos in the [Issues](https://github.com/coinbase/rosetta-ethereum/issues) section with the `help wanted` tag. Be sure to reach out on our [community](https://community.rosetta-api.org) before you tackle anything on this list.
```

Comment:
I am not sure if `help wanted` label is appropriate for this. https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#about-default-labels

Reply:
What label would you prefer to use? Otherwise I can just eliminate the "with the `help wanted` tag" segment.

Reply:
ya I think people can help with any label - so we could just remove the label statement.